source/class/qx/test/compiler/jsdoc/Parser.js

```diff
@@ -0,0 +1,68 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+   * Henner Kollmann 
+
+************************************************************************ */
+
+qx.Class.define("qx.test.tool.utils.Utils", {
```

Comment:
The filename (`qx/test/compiler/jsdoc/Parser.js`) differs from the class name (`qx.test.tool.utils.Utils`)

Reply:
You are right. Was a start to concert the missing tests